code/game/machinery/status_display.dm

```diff
@@ -93,57 +92,29 @@
 
 /// Immediately change the display to the given two lines.
 /obj/machinery/status_display/proc/update_display(line1, line2)
+	if( \
+		(message1_overlay && message1_overlay.message == line1) && \
+		(message2_overlay && message2_overlay.message == line2))
```

Comment:
```suggestion
	if( \
		(message1_overlay && message1_overlay.message == line1) && \
		&& (message2_overlay && message2_overlay.message == line2) \
	)
```

This is the standard we have for multi-lining as it:

A. Makes it clear when the condition ends
B. Lets conditions be re-ordered and read more easily